### PR TITLE
Issue1373

### LINKED ISSUE #1373: multi-layering does not work for servers lacking an init-time (i.e. most non-MSS servers)

### PR DESCRIPTION
**Pull request by joernu76**

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please send this to stable

File: mslib/msui/multilayers.py [hunk 1]
```diff
@@ -197,10 +197,13 @@ def reload_sync(self):
         if self.synced_reference.itime not in self.synced_reference.itimes:
             self.synced_reference.itime = itimes[-1] if itimes else None
 
-        if self.synced_reference.vtime not in self.synced_reference.vtimes or \
-                self.synced_reference.vtime < self.synced_reference.itime:
-            self.synced_reference.vtime = next((vtime for vtime in vtimes if
-                                                vtime >= self.synced_reference.itime), None) if vtimes else None
+        if self.synced_reference.itime is not None:
+            if (self.synced_reference.vtime not in self.synced_reference.vtimes or
+                    self.synced_reference.vtime < self.synced_reference.itime):
```
Comment (ReimarBauer) [inline mslib/msui/multilayers.py]: for me this is not easy to read. I think it gets better if there is a comment describing this

Comment (joernu76) [reply mslib/msui/multilayers.py]: I tried to make it more readable by reducing visual clutter and the ternary expressions.

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
I'm not sure if this is a new or related problem, I crashed this branch by

multilayer

please have a look on this.

**Comment by joernu76** [comment]
nice catch. one "s" missing

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**